src/pages/public_registration/action_accordions/SupportAccordion.tsx

```diff
@@ -70,26 +79,41 @@ export const SupportAccordion = ({
         {supportTicket && ` - ${t(`my_page.messages.ticket_types.${supportTicket.status}`)}`}
       </AccordionSummary>
       <AccordionDetails sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
-        <Typography>{t('my_page.messages.contact_curator_if_you_need_assistance')}</Typography>
+        {!isOnTasksPage && <Typography>{t('my_page.messages.contact_curator_if_you_need_assistance')}</Typography>}
         {isRegistrationWizard && <Typography>{t('registration.curator_support_info')}</Typography>}
         {supportTicket && (
           <>
             <TicketAssignee ticket={supportTicket} refetchTickets={refetchData} />
-            {userIsCurator &&
-              window.location.pathname.startsWith(UrlPathTemplate.TasksDialogue) &&
-              supportTicket.status !== 'Completed' && (
-                <LoadingButton
-                  sx={{
-                    alignSelf: 'end',
-                    width: 'fit-content',
-                    bgcolor: 'white',
-                  }}
-                  loading={ticketMutation.isPending}
+            {userIsCurator && isOnTasksPage && supportTicket.status !== 'Completed' && (
+              <LoadingButton
+                sx={{
+                  alignSelf: 'center',
+                  width: 'fit-content',
+                  bgcolor: 'white',
+                }}
+                loading={ticketMutation.isPending}
+                variant="outlined"
+                onClick={() => ticketMutation.mutate({ status: 'Completed' })}>
+                {t('my_page.messages.mark_as_completed')}
+              </LoadingButton>
+            )}
+
+            {tabErrors && (
+              <div>
```

Comment:
Om vi går for denne løsningen burde nok dette flyttes inn til en egen komponent, som kan gjenbrukes i `PublishingAccordion`